config/routes.rb

```diff
@@ -2,6 +2,7 @@
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
   root to: "main#index"
+  get 'search', to: 'search#search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.